File: Day 77/homework/hw3.py
# 3) დაწერეთ ფუნქცია, რომელიც მიიღებს ორ არგუმენტს — ტექსტს და ასოს — და დააბრუნებს, რამდენჯერ გვხვდება ეს ასო ტექსტში ( გაითვალისწინეთ case სენსიტიურობა) 


def count(text, letter):
    if letter.islower():
        return text.count(letter)
    elif letter.isupper():
        return text.count(letter)
    


print(count("Banana", "a"))

File: Day 109/homework/hw3.js
// 3) შექმენით რიცხვების სია და დაითვალეთ მხოლოდ დადებითი რიცხვების ჯამი. შედეგი გამოიტანეთ კონსოლში.


const nums = [12, -19, 193, 187, -156789, 1973]

let count = 0

for(let i = 0; i < nums.length; i++){
    if(nums[i] > 0){
        count += nums[i]
        console.log(count)
    }
}